index.js

```diff
@@ -77,24 +158,29 @@ const logPageLoadTime = async (n, url, now) => {
 
 try {
   if (isGhosteryEnabled) {
-    const addon = await downloadAddon(
-      'https://github.com/ghostery/ghostery-extension/releases/download/v8.9.15/ghostery-firefox-v8.9.15.zip',
-    );
+    if (!fs.existsSync(`profiles/withGhostery/${selectedBrowser}/onboarded`)) {
+      let extension = 'moz-extension';
+      let chromeAddonUrl = '';
 
-    await driver.installAddon(addon, true);
-
-    if (!fs.existsSync('profiles/withGhostery/onboarded')) {
       await driver.wait(
         async () => (await driver.getAllWindowHandles()).length === 2,
       );
       console.log('INFO: Ghostery onboarding opened.');
 
+      if (isChromeSelected) {
+        extension = 'chrome-extension';
+        let handles = await driver.getAllWindowHandles();
+        await driver.switchTo().window(handles[1]);
```

Comment:
doesn't it do the same as switchToWindowWithUrl below?

Reply:
In the case of Chrome, we have to retrieve the addonUUID from the URL, which is why I am doing it this way here for this browser.